cmake/generate_edition.cmake

```diff
@@ -90,9 +90,9 @@ macro(generate_edition)
 
             string(PREPEND HASH "${EE_HASH}+")
         endif()
-        string(APPEND CouchbaseLite_C_SOURCE_ID " ${HASH}")
+        string(APPEND CouchbaseLite_C_SOURCE_ID "${HASH}")
```

Comment:
Technically there is no reason to do an append here anymore, it could be a simple set